src/renderer/gdi/paint.cpp

```diff
@@ -543,20 +543,24 @@ bool GdiEngine::FontHasWesternScript(HDC hdc)
     };
     const auto DrawCurlyLine = [&](const auto x, const auto y, const auto cCurlyLines) {

```

Comment:
BTW no that I see this, it may make sense to not use `auto` here, but rather explicit types. That way we ensure that we don't get multiple template instantiations.